packages/testing/src/math.cairo

```diff
@@ -0,0 +1,16 @@
+use core::num::traits::ops::overflowing::{OverflowingAdd, OverflowingSub, OverflowingMul};
```

Comment:
I don't think we should export these functions as part of our external API in testing. We should move it to test_common (internal helpers).

Reply:
Moved to openzeppelin_test_common